tests/integration-tests/tests/update/test_update/test_multi_az_create_and_update/pcluster_update_1.config.yaml

```diff
@@ -31,5 +31,6 @@ Scheduling:
           MaxCount: 4
       Networking:
         SubnetIds:
-          - {{ public_subnet_ids[2] }}
+          # us-isob-east-1 provides 2 availability zones.
+          - {% if region == "us-isob-east-1" %} {{ public_subnet_ids[0] }} {% else %} {{ public_subnet_ids[2] }} {% endif %}
```

Comment:
Nice to have: The condition should be if a region only has two azs, rather than a hard-coded us-isob-east-1

Reply:
I agree, this is just a quick&dirty fix to unblock the testing, but since it seems that we may have more time we could also improve